pkgs/development/python-modules/pyunormalize/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "pyunormalize";
+  version = "15.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-z0qHRRoPHLdpEaqX9DL0V54fVkovDITOSIxzpzkBtsE=";
+  };
+
+  # do not run tests
+  doCheck = false;
```

Comment:
Upstream repo seem to contain [tests](https://github.com/mlodewijck/pyunormalize/tree/master/pyunormalize/test).

Please add `pythonImportsCheck`.

Reply:
Done please check everything again.